src/uint256.h

```diff
@@ -127,6 +150,7 @@ class uint160 : public base_blob<160> {
 class uint256 : public base_blob<256> {
 public:
     static std::optional<uint256> FromHex(std::string_view str) { return detail::FromHex<uint256>(str); }
+    consteval explicit uint256(std::string_view hex_str) : base_blob<256>(hex_str) {}
```

Comment:
Attempted doing:
```C++
using base_blob<256>::base_blob;
```
and same for `uint160`.

But ran into GCC issues with inherited consteval constructors (didn't try other compilers): https://gcc.gnu.org/bugzilla/show_bug.cgi?id=106084

Tried adding
```C++
consteval explicit uint160(std::string_view hex_str)
```
And replacing
```C++
    BOOST_CHECK_EQUAL(HexStr(cc1.out.scriptPubKey), HexStr(GetScriptForDestination(PKHash(uint160(ParseHex("816115944e077fe7c803cfa57f29b36bf87c1d35"))))));
```
with
```C++
    BOOST_CHECK_EQUAL(HexStr(cc1.out.scriptPubKey), HexStr(GetScriptForDestination(PKHash(uint160{"816115944e077fe7c803cfa57f29b36bf87c1d35"})))));
```
But test failed because the bytes are reversed. So holding off on touching `uint160`.